code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/ECIDTests.java

```diff
@@ -112,15 +111,15 @@ public void testECID_equals() {
 		ECID a = new ECID();
 		ECID b = new ECID(a.toString());
 
-		assertTrue(a.equals(b));
-		assertTrue(b.equals(a));
-		assertTrue(a.equals(a));
-		assertTrue(b.equals(b));
+		Assert.assertEquals(a, b);
```

Comment:
Let's import
`import static org.junit.Assert.assertEquals`; 
like assertNotEquals and other assert methods.